src/motion/magic/Magic.tsx

```diff
@@ -207,7 +217,13 @@ export class Magic extends React.Component<FunctionalProps> {
         const scaleX = values.get("scaleX", 1)
         const scaleY = values.get("scaleY", 1)
         const rotate = values.get("rotate", 0)
-        const borderRadius = values.get("borderRadius", "")
+        const borderTopLeftRadius = values.get("borderTopLeftRadius", "")
+        const borderTopRightRadius = values.get("borderTopRightRadius", "")
+        const borderBottomLeftRadius = values.get("borderBottomLeftRadius", "")
+        const borderBottomRightRadius = values.get(
+            "borderBottomRightRadius",
+            ""
+        )
```

Comment:
Similarly to how we could combine the `style` values for individual border radii into a single prop, we could take a similar approach here, and get and set a single value for border-radius with all 4 values to simplify this down to a single get/set 🤔 

I'm not sure it would necessarily simplify the code over all, but it would make it more compact by each property type being handled in a singular way.

Reply:
I was under the understanding that `border` can only set each corner to a single radius if its setting four values, whereas to correct for each axis we need to set each corner to two values. I think its fine like this for now though.

Reply:
Ahh you might be right. Great 👍